### ISSUE TITLE
Run dependence for condition queries

### ISSUE THREAD

**Issue by sdobbs** [JeffersonLab/rcdb#57]
We need to be able to specify some run-dependent conditions for standard search aliases e.g. "@is_production", so it would be nice if these conditions could be built into the aliases, or some other method.

**Comment by DraTeots** [comment]
Could you suggest real-life example of what and how should be searched?

**Comment by sdobbs** [comment]
This is motivated by the idea that at some point the name of the default config file name may change, so one could do something like:

@is_production = ( (run<33000) && (run_config== FCAL_BCAL_PS_m9.conf) ) || ( (run>=33000) && (run_config== FCAL_BCAL_PS_ST_m9.conf) )

In reality, the search would be a lot more complex than this.

At some point, daq_run might also be set to a non-"EXPERT" value.

**Comment by DraTeots** [comment]
Aha. Now I clearly understand the problem. Could inventing one more condition that track this condition fix this? For example run_config_xxx which depends on run_config but is the same for run<33000 and run>=33000

**Comment by sdobbs** [comment]
Yes... interesting... I think that could work?  Let me think about this.
